multihost_test/Bugzillas/test_subid_ranges_configurable.py

```diff
@@ -0,0 +1,295 @@
+        :id: 6ff93666-00c0-11ec-9be4-845cf3eff344
+        :bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1859252
+        :steps:
+            1. Define user range in /etc/subuid
+            2. Define group range in /etc/subgid
+            3. Create a process for user
+            4. Check newuidmap command is working
+            with no change in /etc/nsswitch.conf
+            5. Check newgidmap command is working
+            with no change in /etc/nsswitch.conf
+        :expectedresults:
+            1. Should succeed
+            2. Should succeed
+            3. Should succeed
+            4. Should succeed
+            5. Should succeed
+        """
+        execute_cmd(multihost, "echo 'local_anuj:165536:65536' "
+                               "> /etc/subuid")
+        execute_cmd(multihost, "echo 'local_anuj:165536:65536' "
+                               "> /etc/subgid")
+        user = 'local_anuj'
+        ssh1 = SSHClient(multihost.client[0].sys_hostname,
+                         username=user, password='Secret123')
+        (results1, results2, results3) = ssh1.exec_command('unshare -U bash')
+        find_id = "ps -ef | grep bash | grep local_| tail -1"
+        proces_id = [int(word) for word in execute_cmd(multihost,
+                                                       find_id).stdout_text.split()
+                     if word.isdigit()][0]
+        ssh1.exec_command(f"newuidmap {proces_id} 165536 165537 1")
+        ssh1.exec_command(f"newgidmap {proces_id} 165536 165537 1")
+        result = execute_cmd(multihost, f"cat /proc/{proces_id}/uid_map")
+        assert '165536' in result.stdout_text
+        assert f'165537' in result.stdout_text
+        result = execute_cmd(multihost, f"cat /proc/{proces_id}/gid_map")
+        assert '165536' in result.stdout_text
+        assert f'165537' in result.stdout_text
+        execute_cmd(multihost, f'kill -9 {proces_id}')
+        ssh1.close()
+
+    def test_subid_files_nsswitch(self, multihost,
+                                  create_backup,
+                                  create_localuser):
+        """
+        :Title: support pluggable data sources for
+        subid ranges configurable via /etc/nsswitch.conf
+        :id: f6ea2d42-00c0-11ec-b6af-845cf3eff344
+        :bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1859252
+        :steps:
+            1. Define user range in /etc/subuid
+            2. Define group range in /etc/subgid
+            3. Create a process for user
+            4. Check newuidmap command is working
+            with subid: files in /etc/nsswitch.conf
+            5. Check newgidmap command is working
+            with subid: files in /etc/nsswitch.conf
+        :expectedresults:
+            1. Should succeed
+            2. Should succeed
+            3. Should succeed
+            4. Should succeed
+            5. Should succeed
+        """
+        execute_cmd(multihost, "echo 'local_anuj:165536:65536'"
```

Comment:
Test subid range is copy&pasted several times. Please use global const(s).

Reply:
done